### PR TITLE
updating student based routes to be found based on email

### PR DESCRIPTION
**Pull request by melyxlin**
- changed routes "studentInfo" and "studentCourses" to filter student table by email instead of student id.

### CODE REVIEW

File: backend/models.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 class Student(db.Model):
 	__tablename__ = 'student'
 	id = db.Column(db.Integer(), primary_key=True)
-	studentId = db.Column(db.Integer())
```
Comment (bianca-sparxs) [inline backend/models.py:6]: When I request '/students' sever returns:
AttributeError: 'Student' object has no attribute 'studentId'

**Review by bianca-sparxs — CHANGES_REQUESTED** [review]
I think you took out studentId in the model but the student table still has column studentId

File: backend/models.py (hunk 1 above)
Comment (bianca-sparxs) [reply backend/models.py:6]: Otherwise rest of the routes look gud

Comment (melyxlin) [reply backend/models.py:6]: Fixed in latest push.

**Review by bianca-sparxs — APPROVED** [review]
*chef's kiss* fantastico!

**Pull request merged**